StopDetailHeading.tsx

```diff
@@ -1,9 +1,33 @@
 import React from 'react';
+import PopoverMenu from './PopoverMenu';
 
 interface StopDetailHeadingProps {
   name: string;
+  stopId: number;
+  onStopAmenitiesClick: () => void;
 }  
 
-function StopDetailHeading({ name }: StopDetailHeadingProps) {
-  return <h2>{name}</h2>
+function StopDetailHeading({ name, stopId, onStopAmenitiesClick }: StopDetailHeadingProps) {
+  
+  const menuItems = [
+    {
+      name: 'Stop Schedule',
```

Comment:
since we added a new property titled "type", to distinguish which one is a button and a link, we will need to add `type: "link"`